Events/app/src/main/res/layout/onboarding_4.xml

```diff
@@ -44,25 +44,29 @@
 		android:id="@+id/tagRecycler"
 		android:layout_width="0dp"
 		android:layout_height="0dp"
-		android:layout_marginBottom="@dimen/spacing_3l"
+		android:layout_marginBottom="@dimen/spacing_xl"
 		android:layout_marginTop="@dimen/spacing_xl"
 		android:orientation="vertical"
+		android:paddingBottom="@dimen/spacing_5l"
+		android:clipToPadding="false"
+		android:paddingLeft="1dp"
```

Comment:
?

Reply:
the leftmost items in the recycler view were slightly cut off on the left without the left padding